index.html

```diff
@@ -0,0 +1,865 @@
+                  <img src="https://placekitten.com/200/286" class="rounded-circle mr-2" width="70" height="65"
+                    alt="kitty" />
+                  <div class="d-flex flex-column justify-content-center">
+                    <p class="font-weight-bolder m-0">Feed All The Kittens Milk Now</p>
+                    <p class="m-0">
+                      <span class="text-uppercase text-secondary">By</span>
+                      <span class="font-weight-bold text-secondary">THE NEKO MASTER</span>
+                    </p>
+                  </div>
+                </div>
+                <span
+                  class="p-1 font-weight-bold bg-dark rounded-circle text-center text-white position-absolute centering-absolute">
+                  VS
+                </span>
+                <div class="d-flex flex-row border-bottom-manual py-3">
+                  <div class="d-flex flex-column justify-content-center">
+                    <p class="font-weight-bolder m-0">Feed All The Kittens Tuna Now</p>
+                    <p class="m-0">
+                      <span class="text-uppercase text-secondary">By</span>
+                      <span class="font-weight-bold text-secondary">THE NEKO MASTER</span>
+                    </p>
+                  </div>
+                  <img src="https://placekitten.com/200/285" class="rounded-circle ml-auto" width="70" height="65"
+                    alt="kitty" />
+                </div>
+              </div>
+            </div>
+            <!-- Opinion -->
+            <div class=" bg-white col-12 my-3 pb-5">
+              <h6 class="h6 text-left text-secondary">OPINION</h6>
+              <div class="d-flex flex-row justify-content-start border-bottom my-2 py-2">
+                <img src="https://placekitten.com/200/283" class="rounded-circle mr-2" width="70" height="65"
+                  alt="kitty" />
+                <div class="d-flex flex-column justify-content-center">
+                  <p class="h6 font-weight-bolder">Can Kittens Rule The World Again?</p>
+                  <p class="small font-weight-bolder text-secondary">
+                    ASKS THE NEKO MASTER
+                  </p>
+                </div>
+              </div>
+              <div class="d-flex flex-row justify-content-start border-bottom my-2 py-2">
+                <img src="https://placekitten.com/200/283" class="rounded-circle mr-2" width="70" height="65"
+                  alt="kitty" />
+                <div class="d-flex flex-column justify-content-center">
+                  <p class="h6 font-weight-bolder text-capitalize">
+                    Why the cats are domestic? to submit humans to their masterplan?
+                  </p>
+                  <p class="small font-weight-bolder text-secondary">
+                    ASKS THE NEKO MASTER
+                  </p>
+                </div>
+              </div>
```

Comment:
fix this take a look
![image](https://user-images.githubusercontent.com/51195150/99135025-b0166a00-2620-11eb-9c9b-b74f799eea4f.png)
)

Reply:
This is not appearing on my screen nor on my coding partner's screen!